omnistrate_tests/test_operator_out_of_memory.py

```
@@ -0,0 +1,245 @@
+import sys
+import signal
+from random import randbytes
+from pathlib import Path
+import threading
```

Ruff (0.11.9)

`threading` imported but unused

Remove unused import: `threading`

(F401)